tests/runtime/azure/test_azure_remote_qir.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2024 qBraid
+#
+# This file is part of the qBraid-SDK
+#
+# The qBraid-SDK is free software released under the GNU General Public License v3
+# or later. You can redistribute and/or modify it under the terms of the GPL v3.
+# See the LICENSE file in the project root or <https://www.gnu.org/licenses/gpl-3.0.html>.
+#
+# THERE IS NO WARRANTY for the qBraid-SDK, as per Section 15 of the GPL v3.
+
+"""
+Unit tests for Azure Quantum runtime (remote) for QIR
+
+"""
+
+from __future__ import annotations
+
+import pytest
+from azure.quantum.target.microsoft import MicrosoftEstimatorResult
+from qiskit import QuantumCircuit
+
+from qbraid.runtime import DeviceStatus, JobStatus
+from qbraid.runtime.azure import AzureQuantumProvider
+from qbraid.transpiler.conversions.qiskit import qiskit_to_pyqir
+
+pytest.importorskip("pyqir")
+import pyqir
```

Comment:
`pytest.importskip(<module>)` returns the imported module. e.g.

```python
qiskit = pytest.importskip("qiskit")

circuit = qiskit.QuantumCircuit(1)
```

Reply:
See the other `importorskip` response